healthcare/README.md

```diff
@@ -0,0 +1,94 @@
+## Explore the World
```

Comment:
The name should be updated here. It can be something like "QA Application for Healthcare"